### DISCUSSION TITLE
"Unreleased" vs "Released"

### DISCUSSION THREAD

**Discussion by thomoe** [quay/clair#2114] [Q&A] [answered]
Hi,

When I check the Red Hat release noted, it seems Red Hat uses version [4.5.1](https://github.com/quay/clair/releases/tag/v4.5.1) for quay image 3.8.15, [4.7.2](https://github.com/quay/clair/releases/tag/v4.7.2) for quay 3.10.6 and [4.7.4](https://github.com/quay/clair/releases/tag/v4.7.4) for 3.12. All these clair versions are marked "Unreleased" in the release notes here on github, while e.g. [4.7.3](https://github.com/quay/clair/releases/tag/v4.7.3) is not tagged "Unreleased" - and apparently not used Red Hat.

What are the recommendations? Should I avoid the "unreleased" version, or is the higher patch version generally  considered safer, even if the release notes say "Unreleased"?

**Comment by hdonnay** [comment, marked as answer]
I think what you're pointing out is just an artifact of the tool that builds the release notes. All our releases are released.

**Reply by thomoe** [reply]
Thanks, just wanted to make sure I'm not missing anything there.

**Comment by frostmar** [comment]
As a consumer reading the release notes, anything marked "unreleased" is very confusing, if this can be fixed it would be helpful
